website/sidebarsIntegrations.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
                     items: [
                         "services/bookstack/index",
                         "services/dokuwiki/index",
+                        "services/espocrm/index",
```

Comment:
```suggestion
                        "services/espoCRM/index",
```

Reply:
messed up the directory naming; as I'm not using an IDE, I simply changed the path rather than change the directory name (I'd rather conform to your lowercase standards; unsure the quickest way to do that without re-doing the PR)